src/index.css

```diff
@@ -1,10 +1,12 @@
-.thebelab-cell {
+.thebelab-cell,
+.thebe-cell {
```

Comment:
All `thebelab-` rule int his file have now been duplicated as `thebe-` classes - allowing any user code depending on `thebelab-` to continue working for a deprecation cycle